### ISSUE TITLE
search role should be set on the form which wraps search input

### ISSUE THREAD

**Issue by alicewriteswrongs** [mitodl/ocw-hugo-themes#379]
If you read the documentation on MDN for `role="search"` it is actually much more narrow than how we are currently (incorrectly) using it: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/search_role

We are using it right now more like a section, to wrap the entire portion of the page containing the search results:

<img width="1570" alt="Screen Shot 2022-01-26 at 10 50 53 AM" src="https://user-images.githubusercontent.com/6207644/151197437-b8d2e335-8eb7-4957-bca2-a66a6804ccb3.png">

Instead, it should be set on a `form` tag which contains an `input` tag with `type="search"`.

- [ ] remove `role="search"` from the `div` that it is currently applied to
- [ ] set `role="search"` on the `form` tag which wraps our search box
- [ ] test out the usage w/ a screenreader on both the home page and the search page
